beetsplug/fetchart.py

```diff
@@ -743,6 +756,14 @@ def get(self, album, plugin, paths):
                                       match=Candidate.MATCH_FALLBACK)
 
 
+class Embedded(EmbeddedArtSource):
```

Comment:
Any particular reason why `EmbeddedArtSource` and `Embedded` are separate classes? Could they just be one thing? I can't imagine having multiple subclasses of `EmbeddedArtSource` as it stands…

Reply:
Mostly was just cargo culting what I saw in `FileSystem(LocalArtSource)`, as LocalArtSource also had just one class that extended it.  But I agree no good reason for this, so I dropped the `EmbeddedArtSource` class and moved all the functionality into `Embedded(ArtSource)`.